pkg/oam/util/helper.go

```diff
@@ -79,6 +79,21 @@ func LocateParentAppConfig(ctx context.Context, client client.Client, oamObject
 	return nil, errors.Errorf(ErrLocateAppConfig)
 }
 
+// FetchScopeDefinition fetch corresponding scopeDefinition given a scope
+func FetchScopeDefinition(ctx context.Context, r client.Reader,
```

Comment:
no test?

Reply:
sorry for overlooking. I will add a test for it.